### PR TITLE
fix: NaNs containing forecast on some ets configs

### LINKED ISSUE #1285: [Bug]: NaN-filled predict from ETSModel operation on metric evaluation
## Expected Behavior
Passed integration test:
https://github.com/aimclub/FEDOT/blob/0bdece11af60d6e9abc84a894ddd66ea960b5611/test/integration/real_applications/test_examples.py#L86-L88

## Current Behavior
Predictions on the metric evaluation process of a ETSModel end up being NaN-containing:
https://github.com/aimclub/FEDOT/blob/0bdece11af60d6e9abc84a894ddd66ea960b5611/fedot/core/operations/evaluation/operation_implementations/models/ts_implementations/statsmodels.py#L276-L283

## Possible Solution
Maybe with a multiplicative trend [these lines in statsmodels](https://github.com/statsmodels/statsmodels/blob/633b1f334adfa0172cf2a74396b1c4716621209f/statsmodels/tsa/exponential_smoothing/ets.py#L1223-L1230) are raising some kind of exception (e.g. zero-devision) with a small enough endog values.

## Steps to Reproduce
You can add the following code in [api_forecasting.py](https://github.com/aimclub/FEDOT/blob/0bdece11af60d6e9abc84a894ddd66ea960b5611/examples/simple/time_series_forecasting/api_forecasting.py#L64):

```py
def run_ts_forecasting_example(dataset='australia', horizon: int = 30, timeout: float = None,
                               visualization=False, validation_blocks=2, with_tuning=True):
    train_data, test_data, label = get_ts_data(dataset, horizon, validation_blocks=validation_blocks)
    # init model for the time series forecasting
    pipeline = Pipeline().load('<PATH_TO_PIPELINE>')
    model = Fedot(problem='ts_forecasting',
                  task_params=Task(TaskTypesEnum.ts_forecasting,
                                   TsForecastingParams(forecast_length=horizon)).task_params,
                  timeout=timeout,
                  n_jobs=-1,
                  metric='mae',
                  with_tuning=with_tuning.
                  initial_assumption=pipeline)
    ...
```
Here is one of a troubled pipelines: [0_pipeline_saved.zip](https://github.com/aimclub/FEDOT/files/15017231/0_pipeline_saved.zip)

### PR DESCRIPTION
**Pull request by Lopa10ko**
This is a 🐛 bug fix.

- [x] add default ETSModel fall-back on a convergence warning
- [x] run local tests on troubled pipelines
- [x] run integration workflow  

closes #1285

### CODE REVIEW

File: fedot/core/operations/evaluation/operation_implementations/models/ts_implementations/statsmodels.py [hunk 1]
```diff
@@ -255,22 +257,33 @@ def __init__(self, params: OperationParameters):
...
-        )
-        self.model = self.model.fit(disp=False)
+        try:
+            with warnings.catch_warnings():
+                warnings.filterwarnings("error", category=ConvergenceWarning)
+                self._init_model(endog)
+                # if convergence warning is caught, switch to default ETSModel
+                self.model.fit(disp=False)
+        except ConvergenceWarning as e:
+            self.params.update(**{'error': 'add', 'trend': None, 'seasonal': None})
```
Comment (nicl-nno) [inline fedot/core/operations/evaluation/operation_implementations/models/ts_implementations/statsmodels.py:282]: Вроде в таком случае надо ещё в model_repository.json к ets добавить тег correct_params

**Review by nicl-nno — APPROVED** [review]

File: fedot/core/operations/evaluation/operation_implementations/models/ts_implementations/statsmodels.py (hunk 1 above)
Comment (Lopa10ko) [reply fedot/core/operations/evaluation/operation_implementations/models/ts_implementations/statsmodels.py:282]: [3493438](https://github.com/aimclub/FEDOT/pull/1292/commits/3493438e9c8ccf3d0b94cf4871cea6083e3f052b)
